Tests/DatadogTests/Datadog/Core/System/CarrierInfoProviderTests.swift

```diff
@@ -9,9 +9,9 @@ import CoreTelephony
 @testable import Datadog
 
 class CarrierInfoProviderTests: XCTestCase {
-    #if os(iOS)
+    #if canImport(CoreTelephony)
```

Comment:
It's kinda imbalanced ☯️, as we do:
```swift
#if os(iOS)
 import CoreTelephony
```
in `CarrierInfoProvider.swift `, no?

Reply:
you are right and `canImport(CoreTelephony)` doesn't work anyway :)
i didn't spend time on it but CI fails due to wrong evaluation of `canImport(CoreTelephony)`, i don't know how because it works in my local 🤷‍♂ 

let me remove them here